lib/improver/tests/wind_downscaling/test_roughnesscorrection.py

```diff
@@ -37,16 +37,60 @@
 import iris
 from iris.coords import AuxCoord
 from iris.tests import IrisTest
+from iris.coord_systems import (LambertAzimuthalEqualArea,
+                                GeogCS)
 import numpy as np
-
-from improver.grids.osgb import OSGBGRID
 from improver.constants import RMDI
 from improver.wind_downscaling import RoughnessCorrection
 
+ELLIPSOID = GeogCS(semi_major_axis=6378137.0,
```

Comment:
Should the definitions of the `ELLIPSOID` and `STANDARD_GRID_CCRS` already in https://github.com/metoppv/improver/blob/1e7827af4e8d2acf7dec9e0dc4ffb8ab485aebe3/lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py#L106 be factored out, so we don't keep redefining this grid?

Reply:
We could do that, sure. Longer term, will this be the best place to define it? It seems a bit hidden away in a unit test script when maybe it should be in a constants file or something.